src/core/comparable.mli

```diff
@@ -5,6 +5,5 @@ module type S = sig
 
   include Equatable.S with type t := t
 
-  val compare:t Comparator.t
-end
-
+  val compare: t Comparator.t
```

Comment:
I am trying to not add b when the module name does not conflict with the standard library. Do you think it's better to have a  b for all the modules?